Hymns/Assets.xcassets/favorites illustration.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "favorites illustration.png",
```

Comment:
Can you rename to Favorites Empty State or Empty Favorites Illustration or something that indicates it's the empty state rather than just "illustration"

Reply:
Resolved in recent commit

Reply:
You didn't actually commit anything

Reply:
Lol, my b